script/Ciel/Day 9/09_01_CIEL09_8.txt

```diff
@@ -379,8 +425,10 @@ Aside from the blood that pooled up on the brick floor, the blood that once dyed
 [sha:9932c7357baa502faefe1ca708e5b54ff7ea333d]{
 -- Page 723, Offset 24013.
 -- 「そっか、先輩はあれぐらいの傷、なんともないんでしたっけ。ヴローヴの時なんてもっと深い傷を受けてたし、並大抵のコトならアイツみたいにすぐに治って―――」
-// lint-off:LintNameMisspellings
-"Ah, I should've know that you could easily shrug off wounds like these. You took far more damage while fighting against Vlov, so I'm sure you'll recover from these run-of-the-mill injuries in no time, just like Arcuei―――"
+// I can see how you could read アイツ as Arcueid here, but I feel like Vlov is more likely?
+"Ah, I should've known that you could easily shrug off wounds like those.
+ You sustained way more grievous wounds during the fight with Vlov,
+ makes sense you'd be able to recover from some scrapes like that just as fast as he di―――"
```

Comment:
I checked the text and he actually does refer to Vlov as アイツ, but if this is Arcueid I'll change it

Reply:
Actually, going over more of this file again I'm still not totally convinced. _Vlov's_ healing capabilities are the ones that keep getting brought up.

Reply:
I still think its Arcueid, because both the other uses of アイツ in this file are comparisons with Ciel. But looking at it more, it certainly is one of the more ambiguous uses.